code/game/machinery/cell_charger.dm

```diff
@@ -3,9 +3,6 @@
 	desc = "It charges power cells."
 	icon = 'icons/obj/power.dmi'
 	icon_state = "ccharger"
-	use_power = IDLE_POWER_USE
-	idle_power_usage = 5
-	active_power_usage = 60
```

Comment:
Is this just because cell chargers pull on well, charge?

Reply:
yes, they consume power to recharge the cells faster/slower depending on parts.
Also as a note, use_power = IDLE_POWER_USE is useless to add because machinery path defaults to it already